staging/src/k8s.io/client-go/rest/request_test.go

```diff
@@ -938,7 +938,10 @@ func TestRequestWatch(t *testing.T) {
 			},
 			Err: true,
 			ErrFn: func(err error) bool {
-				return apierrors.IsInternalError(err)
+				if !strings.Contains(err.Error(), "failed to reset the request body while retrying a request: EOF") {
```

Comment:
I think we *may* need to assert that this is `!apierrors.IsInternalError(err)`, or assert this is not a server error at all (possibly by checking what the type is after unwrapping).